Fails to build on Mac

```
main.go:36:2: build constraints exclude all Go files in
/Users/xyz/f8a/mercator-go/src/github.com/fabric8-analytics/mercator-go/core/fileutils
```

```
# ls core/fileutils/
fileutils_linux.go  fileutils_windows.go
```

so we probably need also a `fileutils_mac.go` ?

CC @rootAvish

--- a/core/fileutils/fileutils_darwin.go
+++ b/core/fileutils/fileutils_darwin.go
@@ -1,0 +1,21 @@
+/*
+ Copyright 2016 Red Hat, Inc.
+
+ Mercator is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Lesser General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+ Mercator is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Lesser General Public License for more details.
+ You should have received a copy of the GNU Lesser General Public License
+ along with Mercator. If not, see <http://www.gnu.org/licenses/>.
+*/
+package fileutils
+
+import "strings"
+
+func IsHidden(path string) (bool, error) {
+	return strings.Contains(path, "/."), nil
+}

